### DISCUSSION TITLE
How to auto confirm screen recording

### DISCUSSION THREAD

**Discussion by tarasov1503** [AndreyPavlenko/Fermata#513] [General]
Andrey, I know you've posted the adb codes to make the app automatically screen record without having to confirm..but I just got a new phone and for the life of me cannot find it anywhere .. can you please repost it when you get a chance? Would be very much appreciated, my phone for some reason keeps glitching out and won't let me accept when it asks to screen record/mirror

**Comment by AndreyPavlenko** [comment]
You can find it here - https://github.com/AndreyPavlenko/Fermata/discussions/432 :+1: 
```
adb shell appops set me.aap.fermata.auto.dear.google.why PROJECT_MEDIA allow
```
